homeassistant/components/lcn/config_flow.py

```diff
@@ -15,16 +16,15 @@
 )
 from homeassistant.core import HomeAssistant
 from homeassistant.data_entry_flow import FlowResult
-from homeassistant.helpers import device_registry as dr, entity_registry as er
-from homeassistant.helpers.typing import ConfigType
 
 from .const import CONF_DIM_MODE, CONF_SK_NUM_TRIES, DOMAIN
+from .helpers import purge_device_registry, purge_entity_registry
 
 _LOGGER = logging.getLogger(__name__)
 
 
 def get_config_entry(
-    hass: HomeAssistant, data: ConfigType
+    hass: HomeAssistant, data: dict[str, Any]
```

Comment:
Is there a reason for the type hint changing here?

Reply:
I remember that I got some strange typing problems with mypy complaining about the `ConfigType`. However, it disappeared for some reason. I reverted the type hint changes.